tests/integration/src/test/java/org/mule/test/integration/exceptions/RollbackExceptionStrategyRedeliveryCountTestCase.java

```diff
@@ -34,6 +34,7 @@ protected String getConfigFile()
     @Test
     public void testRollbackExceptionStrategyNumberOfRetries() throws Exception
     {
+        System.setProperty("RollBackExceptionStrategy", "true");
```

Comment:
Make sure to set this as false in a @after, because if not other test might fail depending on the order of execution.